cypress/features/regression/designSystem/toastStackedDelayed.feature

```diff
@@ -0,0 +1,12 @@
+Feature: Toast Design System component stacked delayed story
```

Comment:
The file is called `toastStackedDelayed` - but you are opening default_story

Reply:
@damianOfKyan you can remove this file and move this code to the `designSystem/toast.feature`. You should use `id` to locate `stacked delayed Toast`. You should put the `ID` in `\cypress\locators\toast\locators.js`